lib/schema/validate.js

```diff
@@ -296,7 +298,7 @@ const generateErrorReport = ({
     }
 
     // eslint-disable-next-line prefer-const
```

Comment:
Any way to make this a const instead? Better not to use these hints - they're there in eslint for a reason. ;)